spec/support/equipment_model_helper.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module EquipmentModelHelper
+  # traits are the private methods defined in this module
+  # a findable model with a category is created with
+  # traits: [:findable, [:with_category, cat: cat]]
+  def mock_eq_model(traits: [], **attrs)
+    model = instance_spy('EquipmentModel', **attrs)
+    traits.each { |trait, *args| send(trait, model, *args) }
+    model
+  end
+
+  def mock_eq_models(count: 1, traits: [], **attrs)
+    Array.new(count) { mock_eq_model(traits: traits, **attrs) }
+  end
+
+  def stub_collection_methods(models)
+    allow(EquipmentModel).to receive(:all).and_return(models)
+    allow(EquipmentModel).to receive(:where).and_return([])
```

Comment:
It's there because without it, `GET index` fails because of a `where` query in the cart. Those were the first tests I refactored, so I thought that it was going to be necessary throughout the controller. I just reran the tests without it, and only the index tests fail, so I'll just explicitly do this in those tests.